backend/models.py

```diff
@@ -17,12 +17,13 @@ class UserMetrics(EmbeddedDocument):
     works_visited = IntField(default=0)
     works_found = IntField(default=0)
     cities_visited = IntField(default=0)
-    posts_written = IntField(default=0)
+    posts_written = IntField(default=0) # not entirely sure what this represents, might wanna change it to works_created
```

Comment:
We could take these out, they were example metrics from the graphql project. I think the equivalent of `works_created` is `works_found`, so we can rename it to `works_created`

Reply:
works_created sounds good